### ISSUE TITLE
@adeira/relay: incompatible flow settings produce unexpected errors 

### ISSUE THREAD

**Issue by jaroslav-kubicek** [adeira/universe#151]
given I have following flow config:

```
[lints]
all=warn
implicit-inexact-object=off
dynamic-export=off
non-array-spread=off
sketchy-null-bool=off
unclear-type=off
untyped-type-import=off

[strict]
implicit-inexact-object
sketchy-null-bool
unclear-type
untyped-type-import

[options]
include_warnings=true
```

running flow gives me errors for generated files:

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/components/__generated__/ResultQuery.graphql.js:7:13

Redundant argument. This argument doesn't change any lint settings.

      4│  */
      5│
      6│ /* eslint-disable */
      7│ // flowlint untyped-type-import:off
```

**Proposed solution:**

generated files are annotated with `@flow strict`

**EDIT**

based on discussion below, it's not a bug, but we might get better feedback back to the user what should be changed

**Comment by tbergq** [comment]
Try `untyped-type-import=off` -> `untyped-type-import=error`

**Comment by tbergq** [comment]
But yes, we should probably optionally be adding these disables 🤔

**Comment by jaroslav-kubicek** [comment]
yes, one should have `untyped-type-import` turned to warnings at least by default 🙂 

still, I would consider one or more of the following:
- checking whether `.flowconfig` is compatible and recommend changes
- generate files with `@flow strict`
- document this potential error in docs

**Comment by mrtnzlml** [comment]
Can you turn the `untyped-type-import` to `warn`? It's a red alert for you and I don't think it's a good idea to be adding support for codebases ignoring such things (which is big mistake anyway).

**Comment by jaroslav-kubicek** [comment]
yes, I changed it even to error, it was just unexpected and don't know what I would do as person unaware of the flow linting, so I was thinking whether we could give some advice to user

**Comment by mrtnzlml** [comment]
Is this also related to the migration we were talking about somewhere else? We can mention it there as well. You have the differences fresh in your head so feel free to kick it off with few sentences pls.

**Comment by mrtnzlml** [comment]
@jaroslav-kubicek So can we close this or what do you want to do here? I see there are many new issues but we have to be killing them as well... :)

Originally, the Relay project was set in a certain way for Universe applications and the rest of the people had to keep their external projects on the same quality level to avoid any problems. I don't know how to tackle this better.

**Comment by jaroslav-kubicek** [comment]
yes, closing, will be solved by #154
